src/webnn_native/nnapi/NnapiManager.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_NNAPI_MANAGER_H_
+#define WEBNN_NATIVE_NNAPI_MANAGER_H_
+
+#include <map>
+#include <set>
+#include <unordered_set>
+#include <vector>
+
+#include "NeuralNetworksTypes.h"
+#include "NnapiUtils.h"
+#include "nnapi_implementation.h"
+#include "webnn_native/Error.h"
+#include "webnn_native/nnapi/ErrorNnapi.h"
+
+namespace webnn_native { namespace nnapi {
+
+    class NnapiManager {
+      public:
+        explicit NnapiManager();
+
+        ~NnapiManager() {
+            mNnapi->ANeuralNetworksCompilation_free(mNnCompilation);
+            mNnapi->ANeuralNetworksModel_free(mNnModel);
+        }
+
+        MaybeError CreateOperandAndSetMemory(std::string name, NodeInfo* node, void* buffer);
+        size_t SetInputMemory(int32_t index,
+                              const ANeuralNetworksOperandType* type,
+                              const ANeuralNetworksMemory* memory,
+                              size_t offset,
+                              size_t length);
+        size_t SetOutputMemory(int32_t index,
+                               const ANeuralNetworksOperandType* type,
+                               const ANeuralNetworksMemory* memory,
+                               size_t offset,
+                               size_t length);
+        MaybeError CreateScalarOperand(uint32_t type, void* data, uint32_t& index);
+        MaybeError CreateInputOutputOperand(std::string name, NodeInfo* node, bool input = true);
+        MaybeError CreateOperand(NodeInfo* node);
+        MaybeError AddOperation(int32_t opCode,
+                                size_t inputLen,
+                                const uint32_t* input,
+                                size_t outputLen,
+                                const uint32_t* output);
+        MaybeError AddVecOperand(int32_t index, void* buffer, size_t length);
```

Comment:
This function is used to set value for one Operand, how about naming it like SetOperandValue()